client/components/summary/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { PropTypes } from 'react';
+import find from 'lodash/find';
+
+const Summary = ( { overrideFields, formValues, layoutItems, summaryTemplate } ) => {
```

Comment:
It renders the contents of a step (for example, the address) in a text-only format (or at least, in a non-editable format). I can't think on any better name. Maybe `StepSummary`?